test/functional/feature_taproot.py

```diff
@@ -1143,6 +1143,12 @@ def predict_sigops_ratio(n, dummy_size):
         tap = taproot_construct(pubs[0], scripts)
         add_spender(spenders, "alwaysvalid/notsuccessx", tap=tap, leaf="op_success", inputs=[], standard=False, failure={"leaf": "normal"}) # err_msg differs based on opcode
 
+    # == Test case for https://github.com/bitcoin/bitcoin/issues/24765 ==
```

Comment:
Would prefer something more descriptive inline, rather than a link

Reply:
In addition to a link. I think the link is useful for people that want to know further details. But also agree that a short inline description/title would make sense here.